### DISCUSSION TITLE
Is persistQueryClient plugin safe to use with Vue?

### DISCUSSION THREAD

**Discussion by cory-weiner** [TanStack/query#6409] [Q&A]
I would like to persist my query client so the cache is restored between browsing sessions, however I noticed that the persistQueryClient plugin is only listed as part of the React documentation. Is the plugin considered safe to use with Vue?

**Comment by DamianOsipiuk** [comment]
Try fine-grained storage persister
https://tanstack.com/query/latest/docs/framework/vue/examples/persister

**Reply by dcphong** [reply]
it's error bro
